extension_cpp/csrc/sycl/muladd.sycl

```diff
@@ -0,0 +1,189 @@
+#include <c10/xpu/XPUStream.h>
```

Comment:
```suggestion
// Copyright (c) 2025 Intel Corporation

#include <c10/xpu/XPUStream.h>
```

It would also be nice to add a note on the license and we probably would be ok to just reuse any open source license which this project uses... Except there is no license in the top level of this project which I can find:). This is something to be clarified by maintainers. If license will be added to the top level, we can append a reference to it in this source.